en/08_Changelogs/5.4.0.md

```diff
@@ -210,6 +210,52 @@ The `SilverStripe\CMS\Model\SiteTree.DESCRIPTION` localisation key (along with t
+- The [`SiteTree.icon_class`](api:SilverStripe\CMS\Model\SiteTree->icon_class) configuration property has been deprecated. It will be renamed to `cms_icon_class`. This applies to all subclasses of `SiteTree` as well.
+- [`CMSMain::PageList()`](api:SilverStripe\CMS\Controllers\CMSMain::PageList()) has been deprecated. It will be renamed to `RecordList()`.
+- [`CMSMain::PageListSidebar()`](api:SilverStripe\CMS\Controllers\CMSMain::PageListSidebar()) has been deprecated. It will be removed without equivalent functionality.
+- [`CMSMain::LinkPages()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPages()) has been deprecated. Use [`CMSMain::LinkRecords()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecords()) instead.
+- [`CMSMain::LinkPagesWithSearch()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPagesWithSearch()) has been deprecated. Use [`CMSMain::LinkRecordsWithSearch()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecordsWithSearch()) instead.
+- [`CMSMain::LinkPageEdit()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPageEdit()) has been deprecated. Use [`CMSMain::LinkRecordEdit()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecordEdit()) instead.
+- [`CMSMain::LinkPageSettings()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPageSettings()) has been deprecated. Use [`CMSMain::LinkRecordSettings()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecordSettings()) instead.
+- [`CMSMain::LinkPageHistory()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPageHistory()) has been deprecated. Use [`CMSMain::LinkRecordHistory()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecordHistory()) instead.
+- [`CMSMain::LinkPageAdd()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPageAdd()) has been deprecated. Use [`CMSMain::LinkRecordAdd()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkRecordAdd()) instead.
+- [`CMSMain::LinkPreview()`](api:SilverStripe\CMS\Controllers\CMSMain::LinkPreview()) has been deprecated. Use [`SiteTree::CMSEditLink()`](api:SilverStripe\CMS\Model\SiteTree::CMSEditLink()) instead.
+- [`CMSMain::SiteTreeAsUL()`](api:SilverStripe\CMS\Controllers\CMSMain::SiteTreeAsUL()) has been deprecated. Use [`CMSMain::TreeAsUL()`](api:SilverStripe\CMS\Controllers\CMSMain::TreeAsUL()) instead.
+- [`CMSMain::getSiteTreeFor()`](api:SilverStripe\CMS\Controllers\CMSMain::getSiteTreeFor()) has been deprecated. Use [`CMSMain::getTreeFor()`](api:SilverStripe\CMS\Controllers\CMSMain::getTreeFor()) instead.
+- [`CMSMain::CanOrganiseSitetree()`](api:SilverStripe\CMS\Controllers\CMSMain::CanOrganiseSitetree()) has been deprecated. Use [`CMSMain::canOrganiseTree()`](api:SilverStripe\CMS\Controllers\CMSMain::canOrganiseTree()) instead.
+- [`CMSMain::getSearchContext()`](api:SilverStripe\CMS\Controllers\CMSMain::getSearchContext()) has been deprecated. Use [`SiteTree::getDefaultSearchContext()`](api:SilverStripe\CMS\Model\SiteTree::getDefaultSearchContext()) instead.
```

Comment:
Remember to update this based on https://github.com/silverstripe/silverstripe-cms/pull/3036/files#r1877297991